src/main/scala/rx/lang/scala/observables/package.scala

```diff
@@ -24,4 +24,7 @@ package rx.lang.scala
  * in Scala, because we use a pair `(key, observable)` instead of `GroupedObservable`
  * and a pair `(startFunction, observable)` instead of `ConnectableObservable`. 
  */
-package object observables {}
+package object observables {
+  type SyncOnSubscribe[S, +T] = rx.observables.SyncOnSubscribe[S, _ <: T]
+  type AsyncOnSubscribe[S, +T] = rx.observables.AsyncOnSubscribe[S, _ <: T]
```

Comment:
Yes, the variance is correct here. 
An `OnSubscribe[T]` is a function `Subscribe[T] => Unit`, and the left side of the `=>` is a negative position, but being the argument of `Subscriber` is also a negative position, and minus times minus = plus :wink: